python/dgl/graphbolt/dataloader.py

```diff
@@ -121,6 +121,8 @@ def __init__(
         persistent_workers=True,
         overlap_feature_fetch=True,
         overlap_graph_fetch=False,
+        num_cached_edges=0,
+        threshold=1,
```

Comment:
Add python doc for this 2 parameters.

Reply:
Since this is a very high level API, please use clear more descriptive parameter name.
num_cached_edges -> num_gpu_cached_edges
threshold -> gpu_cache_threshold

or better ones.

Similarly for fetch_and_sample method.

Reply:
Changed to your suggested names.